tests/integration_tests/test_matchmaker.py

```diff
@@ -40,6 +42,27 @@ async def test_game_launch_message(lobby_server):
     assert msg2["map_position"] == 2
 
 
+@fast_forward(70)
+async def test_game_launch_message_map_generator(lobby_server):
+    size = 512
+    spawns = 2
+    version = "0.0.0"
```

Comment:
I don't think you actually need the version and seed_match variables. They're only used in the regex so might as well put them directly there for readability. And I might even do the same with option_str, just figure out what it is and hardcode that. That's how I like to do it in tests anyways.